### PR TITLE
move the OCP-10216 to network path

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@lihongan ^^

### CODE REVIEW

File: features/networking/service.feature [hunk 1]
```diff
@@ -532,3 +531,60 @@ Feature: Service related networking scenarios
       | type          | merge                                            |
     Then the step should fail
     And the output should contain "does not completely cover the previous range"
+
+  # @author hongli@redhat.com
```
Comment (lihongan) [inline features/networking/service.feature:535]: please update the author to you as well, that's helpful when someone checking log and contacting the owner directly.

Comment (zhaozhanqi) [reply features/networking/service.feature]: sure if you'd like. updated it

**Comment by lihongan** [comment]
/lgtm
Thank you @zhaozhanqi . I'll remove the case from our feature file.

**Comment by zhaozhanqi** [comment]
> /lgtm
> Thank you @zhaozhanqi . I'll remove the case from our feature file.
FYI, I already removed it in this PR: 
https://github.com/openshift/cucushift/pull/8244/files

**Comment by lihongan** [comment]
> FYI, I already removed it in this PR:
> https://github.com/openshift/cucushift/pull/8244/files

Ahh, no problem. Thank you.

File: features/networking/service.feature [hunk 2]
```diff
@@ -367,9 +367,8 @@ Feature: Service related networking scenarios
   # @author anusaxen@redhat.com
   # @case_id OCP-26035
   @admin
-  Scenario: Idling/Unidling services on OVN
-  Given the env is using "OVNKubernetes" networkType
-  And I have a project
+  Scenario: Idling/Unidling services on sdn/OVN
+  Given I have a project
```
Comment (liangxia) [inline features/networking/service.feature:371]: Indention issue

File: features/networking/service.feature [hunk 3]
```diff
@@ -532,3 +531,60 @@ Feature: Service related networking scenarios
...
+  Scenario: The iptables rules for the service should be DNAT or REDIRECT to node after being idled
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj_name clipboard
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    And I wait until number of replicas match "1" for replicationController "test-rc"
+    Given I use the "test-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :service_ip clipboard
```
Comment (liangxia) [inline features/networking/service.feature:547]: You can use `service.ip` directly.

The same as below steps.

File: features/networking/service.feature (hunk 2 above)
Comment (zhaozhanqi) [reply features/networking/service.feature]: thanks, updated.

File: features/networking/service.feature (hunk 3 above)
Comment (zhaozhanqi) [reply features/networking/service.feature]: updated. thanks

**Pull request merged**